### PR TITLE
update zh translation /reference/command-line-tools-reference/kube-pr…

### PR DESCRIPTION
**Pull request by tanjunchen**

### CODE REVIEW

File: content/zh/docs/reference/command-line-tools-reference/kube-proxy.md [hunk 1]
```diff
@@ -44,448 +44,597 @@ kube-proxy [flags]
...
     <tr>
-    <!--
-      <td></td><td style="line-height: 130%; word-wrap: break-word;">The maximum interval of how often iptables rules are refreshed (e.g. '5s', '1m', '2h22m').  Must be greater than 0.</td>
-    -->
-      <td></td><td style="line-height: 130%; word-wrap: break-word;">iptables 规则刷新的最大时间间隔（例如 '5s'，'1m'，'2h22m'）。必须大于 0。</td>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">
+      <!--
+      The maximum interval of how often iptables rules are refreshed (e.g. '5s', '1m', '2h22m').  Must be greater than 0.
+      -->
+      刷新 iptables 规则的最大间隔（例如 '5s'、'1m'、'2h22m'）。 必须大于0。
```
Comment (yuxiaobo96) [inline content/zh/docs/reference/command-line-tools-reference/kube-proxy.md]:
```suggestion
      刷新 iptables 规则的最大间隔（例如 '5s'、'1m'、'2h22m'）。必须大于0。
```

File: content/zh/docs/reference/command-line-tools-reference/kube-proxy.md [hunk 2]
```diff
@@ -44,448 +44,597 @@ kube-proxy [flags]
...
     <tr>
-    <!--
-      <td></td><td style="line-height: 130%; word-wrap: break-word;">If using the pure iptables proxy, SNAT all traffic sent via Service cluster IPs (this not commonly needed)</td>
-    -->
-      <td></td><td style="line-height: 130%; word-wrap: break-word;">如果使用纯 iptables 代理，SNAT 所有通过服务句群 IP 发送的流量（这通常不需要）</td>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">
+      <!--
+      If using the pure iptables proxy, SNAT all traffic sent via Service cluster IPs (this not commonly needed)
+      -->
+      如果使用纯 iptables 代理，则对通过服务集群 I P发送的所有流量进行 SNAT（通常不需要）
```
Comment (yuxiaobo96) [inline content/zh/docs/reference/command-line-tools-reference/kube-proxy.md]:
```suggestion
      如果使用纯 iptables 代理，则对通过服务集群 IP 发送的所有流量进行 SNAT（通常不需要）
```

**Comment by tanjunchen** [comment]
@yuxiaobo96  Thanks

**Comment by chenrui333** [comment]
Thanks @yuxiaobo96 @tanjunchen !!

**Pull request merged**